### ISSUE TITLE
docs: outdated description of Go version in CONTRIBUTING.md

### ISSUE THREAD

**Issue by yaojingguo** [cockroachdb/cockroach#8329]
Run `make build` now gives the following error message after updating to https://github.com/cockroachdb/cockroach/commit/ce9834efc4b62ca4a77389dbbb7e746068f6ae61:

```
go1\.7.* required (see CONTRIBUTING.md): go version go1.6.2 linux/amd64
make: *** [.go-version] Error 1
```

But [CONTRIBUTING.md](https://github.com/cockroachdb/cockroach/blob/master/CONTRIBUTING.md) still says:

> A Go environment with a 64-bit version of Go 1.6.2.

In the Git commit log, I find:

>    Bump to go1.7rc3

After I upgrade to  go1.7rc3, the error message is gone.

**Comment by giskarda** [comment]
FWIW changing .go-version (while we all wait for a release of 1.7) to 1.6 works (i.e make returned without errors)
```
ok check-style 0.000s
0
[~/src/github.com/cockroachdb/cockroach] ➔  make && echo $?
```
```
diff --git a/.go-version b/.go-version
index 4f776b4..95c8a0a 100644
--- a/.go-version
+++ b/.go-version
@@ -1 +1 @@
-GOVERS = go1\.7.*
+GOVERS = go1\.6.*
```

**Comment by tamird** [comment]
I'll fix this when 1.7 is released.

**Comment by yaojingguo** [comment]
@tamird Go 1.7 is released.
